.github/CODEOWNERS

```
@@ -16,6 +16,11 @@
 # ----------------------------------------------------------------------------
 /files/en-us/    @mdn/yari-content-mdn
 
+# ----------------------------------------------------------------------------
+# GITHUB WORKFLOW OWNER(S)
+# ----------------------------------------------------------------------------
+/.github/   @mdn/yari-content-mdn
```

To be specific to the templates:

```suggestion
/.github/ISSUE_TEMPLATE   @mdn/yari-content-mdn
```

---

I was almost thinking of only change the code owner for [ff-project-issue.md](https://github.com/mdn/content/blob/main/.github/ISSUE_TEMPLATE/ff-project-issue.md), because the other two issue templates are referenced in yari, so the devs might want to know and approve changes. Wdyt?

---

Would this on [line 77](https://github.com/mdn/content/blob/f1d0f645561a3658f896d2e0f0c670686c949492/.github/CODEOWNERS#L77C1-L77C28) take precedence?

`/.github/     @mdn/core-dev`

- If yes, should we be changing that line instead of adding this new section?
- Also consider: `core-yari-content` or `yari-content-mdn`?

---

>Would this on [line 77](https://github.com/mdn/content/blob/f1d0f645561a3658f896d2e0f0c670686c949492/.github/CODEOWNERS#L77C1-L77C28) take precedence?

Yes it does, later rules override previous ones, so this issue template addition has to appear *after*